### PR TITLE
Force small blobs to use PutBlob for all sources

### PR DESCRIPTION
**Pull request by nakulkar-msft**

### CODE REVIEW

File: ste/sender-blockBlobFromURL.go [hunk 1]
```diff
@@ -69,9 +69,8 @@ func (c *urlToBlockBlobCopier) GenerateCopyFunc(id common.ChunkID, blockIndex in
 		setPutListNeed(&c.atomicPutListIndicator, putListNotNeeded)
 		return c.generateCreateEmptyBlob(id)
 	}
-	// Small blobs from all sources will be copied over to destination using PutBlobFromUrl with the exception of files
```
Comment (gapra-msft) [inline ste/sender-blockBlobFromURL.go:72]: Should we add a test that then checks the block list after validation? should be empty I guess?

**Comment by murat8603973081** [comment]
This issue looks interesting - I'll try to provide some solutions and submit a PR.

**Review by siminsavani-msft — APPROVED** [review]

**Review by tasherif-msft — APPROVED** [review]
